test/acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -106,6 +106,8 @@ func TestMeshGatewayDefault(t *testing.T) {
 
 	logger.Log(t, "checking that connection is successful")
```

Comment:
I will revert all changes in this file (mesh_gateway_test.go) after review before merging. This was just to get a failing test quickly.